### ISSUE TITLE
How to change  --metric to other things but not 'loss'?

### ISSUE THREAD

**Issue by sisrfeng** [xingyizhou/CenterNet#785]
> self.parser.add_argument('--metric', default='loss',
Is loss in the paper the only supported metric  so far ?
Since val loss is not so relevant to AP on the val set , will it be better to  replace it with IoU loss ?

**Comment by kuanhungchen** [comment]
IMO, you can try `wh_loss` or `off_loss`.

**Comment by sisrfeng** [comment]
> IMO, you can try `wh_loss` or `off_loss`.

I think wh_loss are off_loss not as good as loss, which contain the former.
Now I save model_best according to AP on val set.

**Comment by YuyangLeng** [comment]
> ```
> > self.parser.add_argument('--metric', default='loss',
> ```
> 
> Is loss in the paper the only supported metric so far ? Since val loss is not so relevant to AP on the val set , will it be better to replace it with IoU loss ?

Have you solved this problem? I met the same thing.

**Comment by sisrfeng** [comment]
I save model_best according to AP on val set.
(need some extra code, about 10-30 lines, If I remember correctly)

**Comment by Dipankar1997161** [comment]
@sisrfeng Hey, it's my first day in training, and I ran around 8 epochs for pose estimation on my custom dataset.
Currently, there are 2 models generated "last.pth" and "best.pth"

At the moment, I have a loss of 1.32, and it's still decreasing. 
According to you, should I retrain using the best model or should I keep this running since the loss is still decreasing?

Your answer is highly appreciated.

**Comment by sisrfeng** [comment]
The author says he always use model_last. You can find in the issues
